package.json

```diff
@@ -59,18 +59,22 @@
     "postcss-loader": "^2.0.6",
     "postcss-smart-import": "^0.7.5",
     "precss": "^2.0.0",
+    "prettysize": "^0.1.0",
     "prop-types": "^15.5.10",
     "rails-erb-loader": "^5.0.2",
     "react": "^15.6.1",
     "react-bootstrap": "^0.31.1",
+    "react-bootstrap-table": "^4.0.0",
     "react-bootstrap-timezone-picker": "^1.0.11",
+    "react-data-grid": "^2.0.2",
```

Comment:
this library is not used in settings react app but we are including it package.json file. Does this large package that is more than 20MB gets packaged to settings app?

Reply:
ask Luka